draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -4084,73 +4090,83 @@ RESPONSE:
    </figure>
    </section>
 
-   <section title="Polling for Changes in Status" anchor="examples-polling-status-changes"><t>
-   uCDN SHOULD use the ETags of collections or triggers
-   when polling for changes in status, as shown in the
-   following examples:</t>
+   <section title="Polling for Changes in Status" anchor="examples-polling-status-changes">
+   <t>
+	uCDN SHOULD use the ETags and/or Last-Modified headers, when polling for changes in trigger collections or status of individual triggers,
+	as shown in the following examples:
+   </t>
 
    <figure><artwork><![CDATA[
 REQUEST:
 
-  GET /triggers/pending HTTP/1.1
+  GET /cit/pending HTTP/1.1
   User-Agent: example-user-agent/0.1
-  Host: dcdn.example.com
+  Host: dcdn.com
   Accept: */*
   If-None-Match: "4331492443626270781"
+  If-Modified-Since: Sun, 27 Oct 2024 08:40:23 GMT
 
 RESPONSE:
   HTTP/1.1 304 Not Modified
   Content-Length: 0
-  Expires: Wed, 04 May 2016 08:49:11 GMT
+  Expires: Sun, 27 Oct 2024 08:49:19 GMT
   Server: example-server/0.1
   ETag: "4331492443626270781"
+  Last-Modified: Sun, 27 Oct 2024 08:49:11 GMT
   Cache-Control: max-age=60
-  Date: Wed, 04 May 2016 08:48:11 GMT
+  Date: Sun, 27 Oct 2024 08:48:19 GMT
   Content-Type: application/cdni; ptype=ci-trigger-collection.v2
 
 REQUEST:
 
-  GET /triggers/0 HTTP/1.1
+  GET /cit/3f2d259d-a980-4742-beeb-9392a58129f5 HTTP/1.1
   User-Agent: example-user-agent/0.1
-  Host: dcdn.example.com
+  Host: dcdn.com
   Accept: */*
   If-None-Match: "6990548174277557683"
+  If-Modified-Since: Sun, 27 Oct 2024 08:49:10 GMT
 
 RESPONSE:
 
   HTTP/1.1 304 Not Modified
   Content-Length: 0
-  Expires: Wed, 04 May 2016 08:49:10 GMT
+  Expires: Sun, 27 Oct 2024 08:49:20 GMT
   Server: example-server/0.1
   ETag: "6990548174277557683"
+  Last-Modified: Sun, 27 Oct 2024 08:49:10 GMT
   Cache-Control: max-age=60
-  Date: Wed, 04 May 2016 08:48:10 GMT
+  Date: Sun, 27 Oct 2024 08:48:20 GMT
   Content-Type: application/cdni; ptype=ci-trigger.v2
+
 ]]></artwork>
    </figure>
    <t>
-   When the trigger processing is complete, the contents of the
-   filtered collections will be updated along with their ETags. For
-   example, when the two example triggers are complete, the
+   When the trigger processing is complete, the contents of the filtered collections will be updated.
+   dCDN SHOULD also update "ETag" and "Last-Modified" response headers when sending the collection representations.
+   dCDN SHOULD also use cache control headers, like "Expires" and "Cache-Control", to indicate how caching of
```

Comment:
Maybe "or" instead of "and"?  Again, either would be sufficient.

Reply:
The revised language around ETag and Last-Modified looks good.
This second comment was switching from "and" to "or" is with regard to "Expires" and "Cache-Control".  Typically, only 1 is needed here, with Cache-Control being the better choice, but either one is sufficient alone.